Tasks/AzureKeyVaultV1/operations/KeyVault.ts

```diff
@@ -192,29 +192,31 @@ export class KeyVault {
             return;
         }
 
+        let secretNameWithoutVerson = secretName.split("/")[0];
+
         let doNotMaskMultilineSecrets = tl.getVariable("SYSTEM_DONOTMASKMULTILINESECRETS");
         if (doNotMaskMultilineSecrets && doNotMaskMultilineSecrets.toUpperCase() === "TRUE") {
-            tl.setVariable(secretName, secretValue, true);
+            tl.setVariable(secretNameWithoutVerson, secretValue, true);
             return;
         }
 
         if (secretValue.indexOf('\n') < 0) {
             // single-line case
-            tl.setVariable(secretName, secretValue, true);
+            tl.setVariable(secretNameWithoutVerson, secretValue, true);
         }
         else {
             // multi-line case
             let strVal = this.tryFlattenJson(secretValue);
             if (strVal) {
                 console.log(util.format("Value of secret %s has been converted to single line.", secretName));
-                tl.setVariable(secretName, strVal, true);
+                tl.setVariable(secretNameWithoutVerson, strVal, true);
             }
             else {
                 let lines = secretValue.split('\n');
                 lines.forEach((line: string, index: number) => {
                     this.trySetSecret(secretName, line);
                 });
-                tl.setVariable(secretName, secretValue, true);
+                tl.setVariable(secretNameWithoutVerson, secretValue, true);
             }
         }
     }
```

Comment:
Also, this could be a breaking change. Given that the earlier one (i.e. `secretName/versionIdentifier` variable) would not available. Perhaps we should add both of them to the environment, for now, to support back compatibility.

Reply:
@bansalaseem @Lovakumar any comments?

Reply:
@ds-ms  Have updated the code to set values against both the names.